pystiche_papers/ulyanov_et_al_2016/_nst.py

```diff
@@ -131,6 +131,7 @@ def stylization(
     transformer: Union[nn.Module, str],
     impl_params: bool = True,
     instance_norm: bool = False,
+    edge_size: int = 256,
```

Comment:
Shouldn't this depend on `impl_params`? By default, I don't think the stylization should resize the image.